src/pyramid_debugtoolbar/toolbar.py

```diff
@@ -200,14 +199,22 @@ def toolbar_tween(request):
         except UnicodeDecodeError as e:
             raise URLDecodeError(e.encoding, e.object, e.start, e.end, e.reason)
 
-        last_proxy_addr = None
-        if request.remote_addr:
-            last_proxy_addr = last_proxy(request.remote_addr)
+        client_addr = None
+        if request.remote_addr and ',' not in request.remote_addr:
+            client_addr = request.remote_addr.strip()
+        elif request.remote_addr is not None:
+            warnings.warn(
+                'pyramid_debugtoolbar has detected a broken middleware box '
```

Comment:
just say proxy here instead of middleware box, I think it's more clear to people